daemon/src/daemon.cpp

```diff
@@ -86,7 +86,7 @@ void* watchdog_thread( void* arg )
 
     CF_logger cf_logger = cf_daemon.cf_logger;
     cf_logger.logger( LOG_INFO, "Start watchdog thread" );
-
+  
```

Comment:
nit - only white space changes in this file, consider reverting whitespace change to keep the git history cleaner.

Reply:
I can revert the whitespace

Reply:
I just looked at the code, I would like to keep the line.